WalletWasabi.Tests/UnitTests/WabiSabi/SerializationTests.cs

```diff
@@ -66,6 +66,7 @@ public void GroupElementCollectionSerialization()
 		Assert.Equal(Generators.Gx1, deserializedGroupElements[1]);
 
 		var deserializedGroupElementVector = JsonConvert.DeserializeObject<GroupElementVector>("[\"02E33C9F3CBE6388A2D3C3ECB12153DB73499928541905D86AAA4FFC01F2763B54\",\"0246253CC926AAB789BAA278AB9A54EDEF455CA2014038E9F84DE312C05A8121CC\"]", converters);
+		Assert.NotNull(deserializedGroupElementVector);
```

Comment:
There was no warning here. Why was this added?

Reply:
![image](https://user-images.githubusercontent.com/45069029/191448554-6b1be024-daaa-4cd5-9f31-b43b5e023628.png)